.changelog/8575.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:improvement
```

Comment:
Should these be called out independently. I saw some other examples in the .changelog directory that split things out fairly fine grained so I went with that here. I would be happen to merge these into a single note though.

Reply:
I don't have a strong opinion either way, I think keeping them split up is fine and makes it slightly more clear to someone reading the changelog. 👍